### ISSUE TITLE
Subtitles keeps loading

### ISSUE THREAD

**Issue by fezzie** [popcorn-official/popcorn-android#828]
Before starting the movie the subtitles keeps hanging on 'loading'

![Screenshot_20221223_170041_Popcorn Time](https://user-images.githubusercontent.com/6862917/209364462-e56e6bbe-d584-43fd-af12-fbb37af90cff.jpg)

**Comment by kiriles90** [comment]
Duplicate of https://github.com/popcorn-official/popcorn-android/issues/802. Closing.
